saved-sites/saved-sites-impl/src/main/res/menu/edit_saved_site_menu.xml

```diff
@@ -19,13 +19,13 @@
     <item
         android:id="@+id/action_delete"
         android:icon="@drawable/ic_trash_24"
-        android:title="@string/delete"
+        android:title="@string/deleteSavedSiteConfirmationDialogDelete"
         android:visible="false"
         app:showAsAction="always" />
     <item
         android:id="@+id/action_confirm_changes"
         android:icon="@drawable/ic_check_24"
-        android:title="@string/dialogConfirmTitle"
+        android:title="@string/editBookmark"
```

Comment:
This was `<string name="dialogConfirmTitle">Confirm</string>` before, is it correct that it has changed to     `<string name="editBookmark">Edit Bookmark</string>`?

Reply:
we don’t show the text anymore, this is only a caption. I didn’t want to create a new string just for this, so between confirm and edit I thought it would work.